### PR TITLE
[AUD-267] Support native notifications panel

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description
Add support for full react native notifications drawer

Will need some more QA passes, especially on the native side, but wanted to get a PR out

### Dragons

Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.
Mobile bounce build

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/rj-aud-267

File: src/containers/notification/store/sagas.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ export const USER_INITIAL_LOAD_COUNT = 9
 
 // Gets the polling interval from remoteconfig
 const getPollingIntervalMs = () => {
-  const pollingInterval = getRemoteVar(IntKeys.NOTIFICATION_POLLING_FREQ_MS)
+  const pollingInterval = 1000 // getRemoteVar(IntKeys.NOTIFICATION_POLLING_FREQ_MS)
```
Comment (jowlee) [inline src/containers/notification/store/sagas.ts:49]: Should we just delete the remote var?

File: src/containers/notification/store/mobileSagas.ts [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+import * as notificationActions from './actions'
+import {
+  getNotificationEntities,
+  getNotificationEntity,
+  getNotificationUser,
+  getNotificationUsers
+} from './selectors'
+import { ConnectedNotification, Notification, NotificationType } from './types'
+
+const USER_LENGTH_LIMIT = 7
```
Comment (jowlee) [inline src/containers/notification/store/mobileSagas.ts:21]: This is the number of users to attach to a notification right? 
Could you add a comment.

**Review by jowlee — APPROVED** [review]
Nice, this wasn't too much code. 
Just navigation logic and message passing of the notifications.

File: src/containers/notification/store/sagas.ts (hunk 1 above)
Comment (raymondjacobson) [reply src/containers/notification/store/sagas.ts]: keeping :)

**Pull request merged**